dick.sh

```diff
@@ -24,7 +24,13 @@ dick_usage() {
   "
 }
 
-LENGTH=5
+read -t 0.01 STDIN_LENGTH
```

Comment:
Why do you need the timeout? Just tested without and tests passed smoothly on Ubuntu Xenial . Are you by any chance using an evil OS?